src/server.js

```diff
@@ -40,11 +43,24 @@ function calculateReward(value, volume) {
     return "Doesn't even fit in an Orca!";
   };
 
+  // If someone pastes in units of millions of ISK, convert to units of ones of ISK.
+  // 20.25 -> 20,250,000
+  if (value < 100000) {
```

Comment:
Change the placeholder text to "Value (Millions of Isk)".  Possibly add some help text for "Type / paste in your cargo value in millions of isk and m3; ensure all hulls are repackaged".

Then change the threshold to a million3

Ideally what the calculator would take is an evepraisal link or inventory copypasta, but I got stuck on ship volumes.  Open to ideas for handling that.